hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -558,6 +558,24 @@
       stage: V2
       tests: []
     }
+    {
+      name: chip_rv_dm_end_to_end_debug
+      desc: '''End to end debug test involving full GDB/OpenOCD/JTAG chain
+
+            - Load and begin executing some known binary
+            - Connect GDB via OpenOCD and JTAG to OpenTitan Earlgrey
+            - Trial the following activies within GDB, ensuring the correct
+              behaviour is seen:
+              - Interrupting and halting execution
+              - Setting and hitting breakpoints
+              - Single stepping
+              - Reading and writing registers and CSRs
```

Comment:
```suggestion
              - Reading and writing all registers and CSRs
```
Or should we focus on a specific subset? (We may have to exclude the `pmpcfg` CSRs due to the issues reported in https://github.com/lowRISC/opentitan/issues/14978#issuecomment-1254893707)